### ISSUE TITLE
Asset filtered out and will not be served: add `Rails.application.config.assets.precompile += %w( bootstrap/glyphicons-halflings-regular.woff2 )` to `config/initializers/assets.rb` and restart your server

### ISSUE THREAD

**Issue by yumikohey** [davetron5000/receta#27]
I am trying to follow your tutorial. However, I stuck here. Even though I follow every single step from your tutorial, I don't get it.

Error:
Asset filtered out and will not be served: add `Rails.application.config.assets.precompile += %w( bootstrap/glyphicons-halflings-regular.woff2 )` to `config/initializers/assets.rb` and restart your server

Could you help me with it?

**Comment by davetron5000** [comment]
sorry for the delay.  If you clone this repo, does it work for you?

In either case, can you share your code?  This stuff is really hard to debug without seeing the entire system you are using.

**Comment by nicholascu** [comment]
@yumikohey @davetron5000   I encountered the same error and am not sure why.  However, following the error message and adding 'Rails.application.config.assets.precompile += %w( bootstrap/glyphicons-halflings-regular.woff2 )' to the config/initializers/assets.rb file solved the problem.

**Comment by davetron5000** [comment]
Yeah, they recently added `.woff2` files and those aren't picked up by one of the other magical incantations.  Will update.
